features/nanostack/FEATURE_NANOSTACK/nanostack-interface/NanostackInterface.cpp

```diff
@@ -439,9 +425,9 @@ void NanostackSocket::event_connnect_closed(socket_callback_t *sock_cb)
     close();
 }
 
-NanostackInterface * NanostackInterface::_ns_interface;
+NanostackInterface *NanostackInterface::_ns_interface;
 
-NanostackInterface * NanostackInterface::get_stack()
+NanostackInterface *NanostackInterface::get_stack()
 {
     nanostack_lock();
```

Comment:
Since you have NanostackLockGuard it might be good to replace all uses of nanostack_lock with it. That way the code can't accidentally leave it locked.